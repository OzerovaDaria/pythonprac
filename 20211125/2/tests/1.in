Äëÿ ïåðåõîäà ê ñëàéäó ñîäåðæàíèÿ ñ äðóãèõ ñëàéäîâ äëÿ ñëîâà «Ñîäåðæàíèå» òàêæå íàñòðàèâàåì ïåðåõîä ê ñòàíèöå èëè îáúåêòó (âñïîìíèâ, ÷òî äëÿ
ñëàéäà ñ ñîäåðæàíèåì çàðåçåðâèðîâàíî èìÿ «toc»). Ïîñëå ýòîãî ìîæíî ïðîâåðèòü
äåéñòâèå ïàíåëè íàâèãàöèè â ðåæèìå äåìîíñòðàöèè.
Èòàê, â èòîãå âñåõ òðóäîâ ïîëó÷åíà ïðåçåíòàöèÿ ñ íàâèãàöèåé ïî ñëàéäàì, ñî
ññûëêàìè íà ëîêàëüíûå äîêóìåíòû è
