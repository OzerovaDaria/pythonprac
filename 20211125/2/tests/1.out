Для перехода к слайду содержания с других слайдов для слова «Содержание» также настраиваем переход к станице или объекту (вспомнив, что для
слайда с содержанием зарезервировано имя «toc»). После этого можно проверить
действие панели навигации в режиме демонстрации.
Итак, в итоге всех трудов получена презентация с навигацией по слайдам, со
ссылками на локальные документы и

